src/hera/workflows/_meta_mixins.py

```diff
@@ -155,7 +150,7 @@ class ModelMapperMixin(BaseMixin):
 
     class ModelMapper:
         def __init__(self, model_path: str, hera_builder: Optional[Callable] = None):
-            self.model_path = None
```

Comment:
This one is to remove mypy errors. Some code got more clear types due to the refactoring, so calling `_get_model_attr` start to raise type errors. So updating the model_path to be `list[str]`, not `optional[list[str]]` to avoid errors. Checked related codes and pretty sure this doesn't change the current behaviors.

Reply:
I defer to @elliotgunton on this change 😄

Reply:
Looks like a sensible fix :shipit: